Packs/CommonScripts/Scripts/IsInternalDomainName/IsInternalDomainName.yml

```diff
@@ -0,0 +1,43 @@
+args:
+- auto: PREDEFINED
+  default: true
+  description: A comma-separated list of main - domains.
+  isArray: true
+  name: domains_to_compare
+  required: true
+  secret: false
+- auto: PREDEFINED
+  default: true
+  description: A comma-separated list of subdomains.
```

Comment:
@ShirleyDenkberg These are the domains we want to test that are a part of the previous argument.